Expands material sheet selection in cargo

atlantiscze opened the pull request:
- Can now order plasteel (20 points for 10 sheets, or 80 points for 50 sheets)
- Can now order osmium-carbide plasteel (25 points for 10 sheets or 100 points for 50 sheets)
- Tweaked price of marble (10-> 15 points for 10 sheets, (5x10) - > 60 points for 50 sheets)
- Plasteel, OCP and Marble are now available in two variants - 10 and 50 sheets. 50 sheet variants are slightly more cost-efficient.

atlantiscze committed:
Expands material sheet selection in cargo
- Can now order plasteel (20 points for 10 sheets, or 80 points for 50 sheets)
- Can now order osmium-carbide plasteel (25 points for 10 sheets or 100 points for 50 sheets)
- Tweaked price of marble (10-> 15 points for 10 sheets, (5x10) - > 60 points for 50 sheets)
- Plasteel, OCP and Marble are now available in two variants - 10 and 50 sheets. 50 sheet variants are slightly more cost-efficient.

atlantiscze committed:
Changelog entry

Techhead0 commented:
Marble here is actually cheaper to buy five crates of ten, since you can send the crates back for a net cost of 50 points, versus 55 net points for a single crate of fifty.
Plasteel is break-even, at 75 net points either way.
OCP is the only one that's actually cheaper to buy in bulk, at 100 net points for five tens, or 95 net points for one fifty.

Aside from the prices, code is all good.

Techhead0 commented on code/modules/materials/material_sheets.dm:
```diff
@@ -204,6 +210,9 @@
 
+/obj/item/stack/material/plasteel/ten
+	amount = 10
+
```
Nitpick: All the other tens are before the fifty in the file. But it's minor and not worth blocking the merge.

Techhead0 replied:
Thanks Github for being weird about which line you move.

Minijar commented:
-1 Removes further need for miners.

ZCaliber commented:
Any chance to also add precious metals at ludicrous prices? I mean, achievable and all, but something really expensive to use all that excess points on.

The idea being that miners are the MUCH faster way to get it, being able to sell phoron and platinum. But also making certain metals viable late-round.

atlantiscze commented:
@Minijar 
The goal of this PR is to provide alternative way of obtaining these materials, which will, however, be quite expensive. Prices are of course debatable, i aim for it to be "Expensive but not impossible to purchase without miners"

Keep in mind that sometimes you either don't have miners around, and you need some plasteel/OCP for repairs or similar things. On-station supply is quite limited, and when it runs out, any alternative - even expensive one - is better than a hole in the hull.

@Techhead0 
As we talked on IRC about it, i'll resolve this. Thanks for pointing it out.

@ZCaliber 
Can do that.

atlantiscze committed:
Additional expansion + price balancing
- Prices of 10-sheet packs now take in account selling price of crates/manifests.
- Moved all materials from engineering category to a new category "Materials" (it was getting a bit too long)
- Added gold, silver, phoron, uranium and diamond sheet crates (10 sheets only) as suggested by ZCaliber in the github PR. These are balanced to be very expensive, so the only real use i can see is when miners can't find a specific resource, but find a lot of phoron/platinum that can be sold.
- Renamed Metal sheets to Steel sheets (which they are everywhere else), while i was touching the relevant things.

atlantiscze commented:
Done.
Also created a relevant forum thread with vote: https://baystation12.net/forums/threads/vote-orderable-materials.4848/

atlantiscze commented:
Vote has been open for a bit over 3 days, and seems to be currently in favor of current state of the PR (with 54% votes for current state and 32% for OCP/plasteel only)
